scipy/stats/tests/test_discrete_basic.py

```diff
@@ -148,6 +148,31 @@ def test_ppf_with_loc(dist, args):
             )
 
 
+@pytest.mark.parametrize('dist, args', distdiscrete)
+def test_isf_with_loc(dist, args):
+    try:
+        distfn = getattr(stats, dist)
+    except TypeError:
+        distfn = dist
+    # check with a negative, no and positive relocation.
+    np.random.seed(1942349)
+    re_locs = [np.random.randint(-10, -1), 0, np.random.randint(1, 10)]
+    _a, _b = distfn.support(*args)
+    for loc in re_locs:
+        expected = _b + loc, _a - 1 + loc
+        res = distfn.isf(0., *args, loc=loc), distfn.isf(1., *args, loc=loc)
+        npt.assert_allclose(expected, res, rtol=1e-14)
```

Comment:
this should also work with asserting equality. expected and res should be integer valued

Reply:
Changed it to assert equality. Thanks!